docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -544,3 +544,50 @@ PUT _cluster/settings
   }
 }
 ----
+
+[discrete]
+==== Number of documents in the index can't exceed [2147483519]
+
+Elasticsearch shards reflect Lucene's underlying https://github.com/apache/lucene/issues/5176[index 
+`MAX_DOC` hard limit] of 2,147,483,519 (`(2^31)-1`) docs. This figure calulates 
```

Comment:
NIT: 2147483519 is correct according to the org.apache.lucene.index.IndexWriter#MAX_DOCS, however that is `(2^31) - 129`